public/documentation/embedding_crosswalk/crosswalk_aar.md

```diff
@@ -114,6 +114,8 @@ Then install it into the local Maven repository:
     }
     ```
 
+    Using mavenLocal() instead of remote repo url to refer local aar.
+
```

Comment:
So should we change the code above this comment:

Repositories {
   Maven {
      url 'https://download.01.org/crosswalk/releases/crosswalk/android/maven2'
   }
}
to:
Repositories {
   mavenLocal()
}
?